src/webui/www/private/views/rss.html

```diff
@@ -697,11 +697,12 @@
         };
 
         const removeItem = (paths) => {
+            const encodedPaths = paths.map((path) => encodeURIComponent(path));
             new MochaUI.Window({
                 id: 'confirmFeedDeletionPage',
                 title: 'QBT_TR(Deletion confirmation)QBT_TR[CONTEXT=RSSWidget]',
                 loadMethod: 'iframe',
-                contentURL: 'confirmfeeddeletion.html?paths=' + encodeURIComponent(paths.join('|')),
+                contentURL: 'confirmfeeddeletion.html?paths=' + encodeURIComponent(encodedPaths.join('|')),
```

Comment:
Since you're encoding the individual paths, do you still need to encode it again here?

Reply:
Yes. If the paths or rules contain '|', encoding the individual paths only once may cause an error wthen split the decoded strings with '|'.